content/libraries/klc_history.adoc

```diff
@@ -5,6 +5,11 @@ url = "/libraries/klc/history/"
 +++
 
 ---
+== 3.0.19 - 2019-05-04
+* Clarify rules regarding NC pins
+   * Clarify when they may be omitted
+   * Clarify that they must be hidden when
```

Comment:
Not correct phrase. Is to missing something at the end or just need wordsmithing?

Reply:
yes something went wrong here. fixed